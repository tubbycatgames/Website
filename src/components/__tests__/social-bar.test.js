import React from "react"
import renderer from "react-test-renderer"

import SocialBar from "../social-bar"

describe("Social Bar", () => {
  it("renders a set of linked up social icons", () => {
    const tree = renderer.create(<SocialBar />)
    expect(tree.toJSON()).toMatchSnapshot()
  })
})
