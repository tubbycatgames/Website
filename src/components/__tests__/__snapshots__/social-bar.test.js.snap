// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social Bar renders a set of linked up social icons 1`] = `
<div
  className="bar"
>
  <a
    href="https://tubbycatgames.itch.io/"
  >
    <img
      alt="Itch Page Icon"
      className="icon"
      src="test-file-stub"
    />
  </a>
  <a
    href="https://gamejolt.com/@tubbycatgames"
  >
    <img
      alt="Game Jolt Page Icon"
      className="icon"
      src="test-file-stub"
    />
  </a>
  <a>
    <img
      alt="GitHub Page Icon"
      className="icon"
      src="test-file-stub"
    />
  </a>
  <a
    href="https://discord.gg/vyh5XY"
  >
    <img
      alt="Discord Social Media Icon"
      className="icon"
      src="test-file-stub"
    />
  </a>
  <a
    href="https://twitter.com/tubbycatgames"
  >
    <img
      alt="Twitter Social Media Icon"
      className="icon"
      src="test-file-stub"
    />
  </a>
  <a
    href="https://instagram.com/tubbycatgames"
  >
    <img
      alt="Instagram Social Media Icon"
      className="icon"
      src="test-file-stub"
    />
  </a>
  <a
    href="https://fb.me/tubbycatgames"
  >
    <img
      alt="Facebook Social Media Icon"
      className="icon"
      src="test-file-stub"
    />
  </a>
  <a
    href="https://www.youtube.com/channel/UC_6l3SdhVjdtTd2DnHK2JbQ"
  >
    <img
      alt="YouTube Channel Icon"
      className="icon"
      src="test-file-stub"
    />
  </a>
</div>
`;
